doc/findinfiles.rst

```diff
@@ -20,13 +19,15 @@ This will allow you to view and navigate through all the occurrences of your sea
 You can expand or collapse the search results to view the matches in each file.
 Clicking on a match will automatically open the file and highlight the line where the text was found.
 
-[GIF showing search and cliking file]
+.. image:: images/find_in_files/find-in-files-search.gif
+   :alt: Spyder Find pane showing search and going to a file
 
 If you want to change the scope of your search, select another directory, project or file in the :guilabel:`Search in` menu.
 The locations that you select for your search will be stored in the list so you can access them easily in the future.
 To erase all of these saved directories, select the :guilabel:`Clear the list` option from the dropdown menu in the :guilabel:`Search in` field.
 
-[GIF showing chosing new directory and directory stored in the list]
+.. image:: images/find_in_files/find-in-files-directory.gif
+   :alt: Spyder Find pane showing chosing new directory and directory stored in the list
 
```

Comment:
```suggestion

|

```

This one is missing the extra break after the image, before the section header, while the other one has it.